src/pages/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router';
+import PageLayout from '../../PageLayout/PageLayout';
+import FilterableList from '../../components/FilterableList/FilterableList';
+import getCitiesData from '../../services/getCitiesData';
+import { minValueLength } from '../../common/constants';
+
+function HomePage() {
+  const [cities, setCities] = useState([]);
+  const [inputValue, setInputValue] = useState('');
+  const history = useHistory();
+
+  const getAllCitiesByName = async event => {
+    setInputValue(event.target.value.toLowerCase());
+
+    if (inputValue.length >= minValueLength) {
+      try {
+        let getDataCities = await getCitiesData(inputValue);
```

Comment:
1) why let instead of const? Are you going to mutate it later?
2) getDataCities - is a name for a function - it is represented by a verb: get something. Whereas in this case you should use a noun: const cities... or const citiesData